### PR TITLE
BACKLOG-17115: Use moonstone header

### PR DESCRIPTION
**Pull request by tdraier**
## JIRA

https://jira.jahia.org/browse/BACKLOG-17115

### CODE REVIEW

File: src/javascript/EditPanel/EditPanelHeader/EditPanelHeader.scss [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+.headerChips {
+    & > * {
+        margin-left: var(--spacing-small);
+    }
+}
+
+.padder {
+    height: var(--spacing-medium);
+}
```
Comment (ffffffelix) [inline src/javascript/EditPanel/EditPanelHeader/EditPanelHeader.scss:30]: It seems that class is not used

File: src/javascript/EditPanel/EditPanelHeader/EditPanelHeader.scss [hunk 2]
```diff
@@ -0,0 +1,58 @@
+.headerLeft {
+    display: flex;
+    flex-grow: 1;
+    align-items: center;
```
Comment (ffffffelix) [inline src/javascript/EditPanel/EditPanelHeader/EditPanelHeader.scss:4]: The layout must be manage by the moonstone component itself. But if you really need it you can use the utilities classe provided by moonstone `.flexRow` `flexFluid` `alignCenter`

File: src/javascript/EditPanel/EditPanelHeader/EditPanelHeader.scss [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    height: var(--spacing-medium);
+}
+
+.mainActions {
+    margin-left: var(--spacing-small);
+}
+
+.menu {
+    height: 38px;
+}
```
Comment (ffffffelix) [inline src/javascript/EditPanel/EditPanelHeader/EditPanelHeader.scss:38]: It seems that class is not used

File: src/javascript/EditPanel/EditPanelHeader/EditPanelHeader.scss [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.headerChips {
+    & > * {
+        margin-left: var(--spacing-small);
+    }
+}
```
Comment (ffffffelix) [inline src/javascript/EditPanel/EditPanelHeader/EditPanelHeader.scss:26]: The moonstone component already manage the spacing between status if you put directly the components inside the `status` props, or you can manage it at the `Headerbadges` component level.

File: src/javascript/EditPanel/EditPanelHeader/EditPanelHeader.scss (hunk 2 above)
Comment (tdraier) [reply src/javascript/EditPanel/EditPanelHeader/EditPanelHeader.scss]: replaced with moonstone classes

File: src/javascript/EditPanel/EditPanelHeader/EditPanelHeader.scss (hunk 4 above)
Comment (tdraier) [reply src/javascript/EditPanel/EditPanelHeader/EditPanelHeader.scss]: this class is actually useless

**Review by ffffffelix — APPROVED** [review]

**Pull request merged**